### PR TITLE
 os-3542 lbaas announcement

### PR DESCRIPTION
**Pull request by mbernhardtSys11**

### CODE REVIEW

File: pages/02.Documentation/07.network/02.lbaas/docs.en.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ Unfortunately it is currently not possible to forward the client IP to the insta
 
 We prepared a simple tutorial that shows [basic usage of LBaaS](../../../03.Tutorials/07.lbaas/docs.en.md).
 
+!!! **Feature availability**
+!!! LBaaS was previously only available in region 'dbl'. It has become also available in region 'cbk' in february 2019.
```
Comment (bpetermannS11) [inline pages/02.Documentation/07.network/02.lbaas/docs.en.md]: February should start with upper-case F

**Comment by mbernhardtSys11** [comment]
fucked up merge request, will try again

File: pages/03.Tutorials/07.lbaas/docs.en.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ This means that load balancing rules can be created directly within OpenStack.
 No dedicated load balancer instance is required to use this service.
 
 !!! **Feature availability**
-!!! LBaaS is currently only available in region 'dbl'. It will also be available in region 'cbk' in the future.
+!!! LBaaS was previously only available in region 'dbl'. It has become also available in region 'cbk' in february 2019.
```
Comment (bpetermannS11) [inline pages/03.Tutorials/07.lbaas/docs.en.md]: february -> February

File: pages/02.Documentation/07.network/02.lbaas/docs.en.md (hunk 1 above)
Comment (mbernhardtSys11) [reply pages/02.Documentation/07.network/02.lbaas/docs.en.md]: indeed, thanks

**Pull request closed without merging**